app/repositories/push_notifications_repository.js

```diff
@@ -29,9 +29,21 @@ export default class PushNotificationsRepository extends BaseRepository {
     const data = { devices };
     if (request.message && !message) {
       const m = request.message;
+      let imgUrl = null;
+      if (m.attachments?.length === 1) {
+        imgUrl = await globalThis.storageClient.getDownloadUrl(
+          m.attachments[0].file_id
+        );
+      } else if (m.attachments) {
+        for (let i = 0; i < m.attachments.length; i++) {
+          m.body += `\nImage`;
```

Comment:
not sure it's a good idea to add `Image` for every pic.

I suggest to make it simple:

- pass `imgUrl ` of first image
- do not modify the body